lib/elixir/lib/enum.ex

```diff
@@ -1436,6 +1436,20 @@ defmodule Enum do
   end
 
   @doc """
+  Invokes `fun` for each element in the collection passing that element and the
+  accumulator `acc` as arguments. `fun`'s return value is expected to be
+  `{:cont, acc}`, return `{:halt, acc} to end the reduction.
+  Returns the accumulator.
+
+  ## Examples
```

Comment:
Use new lines after headers and four spaces for code samples. Please check the other docs in the same module for examples.